README.md

```diff
@@ -34,7 +33,7 @@ When we release a version, we push the following images to Dockerhub:
 1. Major.Minor
 1. Major
 
-In addition, each time the `master` build is green, we push a `<version>-<git_version>` (i.e. `0.2.0-d9494c1`) image to our internal repo.
+We also push the Major.Minor.Build image to our Red Hat registry.
```

Comment:
@orenbm I also took the opportunity to remove the above info and say we also post the Major.Minor.Build version to the RH registry

Reply:
removed. this will be added back in once we get RH approval